radix-engine-toolkit/examples/transactions/account-derive-from-mnemonic/Go/README.md

```diff
@@ -0,0 +1,7 @@
+# Example of account derive from mnemonic in Go
+
+Compatible environment: MacOS aarch64.
```

Comment:
Maybe rephrased it as "Tested on MacOS xx.11 with aarch64" since it supports more than MacOS.